src/utils/azure.ts

```diff
@@ -21,16 +28,16 @@ function parseResourceId(id: string): RegExpMatchArray {
     return matches;
 }
 
-export function getResourceGroupFromId(id: string): string {
-    return parseResourceId(id)[2];
+export function getResourceGroupFromId(id: string, caseInsensitiveCheck: boolean = false): string {
```

Comment:
nit. now that I am reading, I wonder if its better to have `caseSensitive: boolean = true` like that we avoid double negatives. Also I dont think you need the word `Check` in it.